src/assets/i18n/de_DE.json

```diff
@@ -297,6 +298,20 @@
   "account.organization.cost_center_management.create.confirmation": "Die Kostenstelle \"{{0}}\" wurde erstellt.",
   "account.organization.cost_center_management.delete.confirmation": "Die Kostenstelle wurde gelöscht.",
   "account.organization.cost_center_management.update.confirmation": "Die Kostenstelle \"{{0}}\" wurde geändert.",
+  "account.organization.hierarchies": "Struktur des Unternehmens",
+  "account.organization.hierarchies.grouppath.tooltip.close": "Ausblenden",
+  "account.organization.hierarchies.grouppath.tooltip.open": "Alle Gruppen anzeigen",
+  "account.organization.hierarchies.groups.button.add.label": "Gruppe hinzufügen",
+  "account.organization.hierarchies.groups.delete.confirmation": "Die Gruppe mit der ID \"{{0}}\" wurde gelöscht.",
```

Comment:
Use German quotation marks instead?

Reply:
There are currently no plans to change quotation marks in German due to extremely high effort to adjust it everywhere.